server/protocols/webadmin/webadmin-data/src/test/java/org/apache/james/webadmin/routes/GroupsRoutesTest.java

```diff
@@ -370,74 +401,133 @@ public void setUp() throws Exception {
 
         @Test
         public void getMalformedGroupShouldReturnBadRequest() {
-            when()
+            Map<String, Object> errors = when()
                 .get("not-an-address")
             .then()
-                .statusCode(HttpStatus.BAD_REQUEST_400);
+                .statusCode(HttpStatus.BAD_REQUEST_400)
+                .contentType(ContentType.JSON)
+                .extract()
+                .body()
+                .jsonPath()
+                .getMap(".");
+
+            assertThat(errors)
+                .containsEntry("statusCode", 400)
+                .containsEntry("type", "InvalidArgument")
+                .containsEntry("message", "The group is not an email address")
+                .containsEntry("cause", "Out of data at position 1 in 'not-an-address'");
```

Comment:
What does "Out of data at position 1 in 'not-an-address'" mean ?

Reply:
That is the message from throwable, I don't want to refactor it